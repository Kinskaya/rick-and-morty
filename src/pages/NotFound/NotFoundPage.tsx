import * as React from 'react';
import { Link } from 'react-router-dom';

class NotFoundPage extends React.Component {
  render() {
    return (
      <div>
        <h1>404 Page is not found</h1>
        Go <Link to="/">Home</Link>
      </div>
    );
  }
}

export default NotFoundPage;
