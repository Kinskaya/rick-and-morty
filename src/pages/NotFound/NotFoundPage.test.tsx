import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import NotFoundPage from './NotFoundPage';

describe('NotFoundPage component', () => {
  it('render correctly', () => {
    const { asFragment } = render(<NotFoundPage />, { wrapper: BrowserRouter });
    expect(asFragment()).toMatchSnapshot();
  });
});
