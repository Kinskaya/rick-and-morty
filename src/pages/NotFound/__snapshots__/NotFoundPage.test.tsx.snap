// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component render correctly 1`] = `
<DocumentFragment>
  <div>
    <h1>
      404 Page is not found
    </h1>
    Go 
    <a
      href="/"
    >
      Home
    </a>
  </div>
</DocumentFragment>
`;
